googleapiclient/http.py

```diff
@@ -38,11 +38,26 @@
 import os
 import random
 import socket
-import ssl
 import sys
 import time
 import uuid
 
+
+try:
+  import ssl
+except ImportError:
+  class _ssl_SSLError(socket.error):
+    # To be precise, ssl.SSLError only had `library` and `reason` since Python 2.7.9,
```

Comment:
Why attempt compatibility? Why not have the class loudly advertise that it is *not* an `ssl.SSLError`, because the code is executing in a place in which the standard library `ssl` module is not available? If someone has a problem that proceeds as far as handling this exception, why not be as helpful to them as we can be?